### DISCUSSION TITLE
support for formatted text in the terminal

### DISCUSSION THREAD

**Discussion by Didymograptus** [pyscript/pyscript#1990] [Q&A] [answered]
I have some Python scripts that import the rich module https://pypi.org/project/rich/ to printed coloured text to the terminal for highlights.
I've imported it and printed to the terminal in PyScript, but unfortunately it doesn't work.
Is there a way to print coloured text to the terminal in PyScript?

I've also tried Termcolor (supported by Pyodide) without success.

**Comment by Didymograptus** [comment]
Colorama does work. https://pypi.org/project/colorama/

**Comment by Didymograptus** [comment, marked as answer]
Also use of Rich Console API works. This is a neater solution than colorama.

https://rich.readthedocs.io/en/stable/console.html

How to use: 

import rich in pyscript.toml

``` Python
from rich.console import Console
console = Console()

console.print(f"{Fore.YELLOW}Hello World!\n")
```

**Comment by WebReflection** [comment]
The terminal is provided by XTerm and I am afraid if they don't support "this or that" library there is not much we can do if not asking why/how we could support those ... but if I understood correctly your issue is solved with `rich.console`, correct?

**Comment by Didymograptus** [comment]
Yes, `rich.console` works fine.
